public/app/features/expressions/components/ToLabel.tsx

```diff
@@ -0,0 +1,26 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+
+export function ToLabel() {
+  const styles = useStyles2(getStyles);
+  return (
+    <div className={styles.button}>
+      <Trans i18nKey="alerting.threshold.To">TO</Trans>
```

Comment:
nit: I don't think we use capitalization in the keys 🤔

Reply:
fixed!